src/appBar/ButtonAppBar.tsx

```diff
@@ -1,14 +1,26 @@
 import * as React from 'react';
 import { LogInLogOutButton } from 'src/auth/components/LogInLogOutButton';
-import { myLocalStorage } from 'src/services/localStorage';
+import { SignUpButton } from 'src/auth/components/signUpForm/SignUpButton';
 import { SettingsButton } from 'src/settings/SettingsButton';
 
-export default function ButtonAppBar() {
-  const isAuth = myLocalStorage.isAuth();
+interface IButtonAppBar {
+  isSuccess: boolean;
+}
+
+export default function ButtonAppBar({ isSuccess }: IButtonAppBar) {
   return (
-    <div className={'buttonAppContainer'}>
-      {isAuth && <SettingsButton />}
-      <LogInLogOutButton />
+    <div>
+      {isSuccess ? (
+        <div className={'buttonAppContainer'}>
+          <SettingsButton />
+          <LogInLogOutButton />
+        </div>
```

Comment:
we repeat `<div className={'buttonAppContainer'}>` for both cases. This means only its children differ in case `isSuccess` changes the value. So let's put `isSuccess` check inside the `<div className={'buttonAppContainer'}>`

Reply:
changed